### PR TITLE
Wc/creating db2 views

### PR DESCRIPTION
**Pull request by chewnoill**

### CODE REVIEW

File: db/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,74 @@
+FROM postgres:13
+
+ADD ./v11.5.4_linuxx64_client.tar.gz /db2
```
Comment (chewnoill) [inline db/Dockerfile:3]: I uploaded this file to be here:
https://drive.google.com/file/d/1pe56WnWNV5P61jbuL-FL9cP1-rV9AIjx/view?usp=drive_link

Comment (cpjolicoeur) [reply db/Dockerfile:3]: I/we can throw this onto an MT S3 bucket as well if needed

File: db/init/db2_views/02-create-schemas.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+-- NOTICE:  Data type not supported (-2) for column VEND#H
+
+create schema db2_GDSDSFIL;
+IMPORT FOREIGN SCHEMA "GDSDSFIL" FROM SERVER as400 INTO db2_GDSDSFIL;
+
+create schema db2_GDSSYFIL;
+IMPORT FOREIGN SCHEMA "GDSSYFIL" FROM SERVER as400 INTO db2_GDSSYFIL;
+
+create schema db2_JVFIL;
+IMPORT FOREIGN SCHEMA "JVFIL" FROM SERVER as400 INTO db2_JVFIL;
```
Comment (chewnoill) [inline db/init/db2_views/02-create-schemas.sql]: importing this schema fails with an error

> ERROR:  tables can have at most 1600 columns
> CONTEXT:  importing foreign table “FIELDREF”

File: db/init/db2_views/02-create-schemas.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+create schema db2_GDSDSFIL;
+IMPORT FOREIGN SCHEMA "GDSDSFIL" FROM SERVER as400 INTO db2_GDSDSFIL;
+
+create schema db2_GDSSYFIL;
+IMPORT FOREIGN SCHEMA "GDSSYFIL" FROM SERVER as400 INTO db2_GDSSYFIL;
+
+create schema db2_JVFIL;
+IMPORT FOREIGN SCHEMA "JVFIL" EXCEPT (
+ "FIELDREF" -- this is a table with more then 1600 columns that postgres won't import
+) FROM SERVER as400 INTO db2_JVFIL;  -- long time +500s
```
Comment (chewnoill) [inline db/init/db2_views/02-create-schemas.sql:19]: JVFIL schema has 6900 tables in it, which might explain why it takes so long to load, might be worth itemizing just the 40 we need?

**Comment by chewnoill** [comment]
superseeded by https://github.com/havandenberg/jv-system/pull/3

**Pull request closed without merging**